contracts/Main.sol

```diff
@@ -34,24 +34,27 @@ contract Main {
         // Migrate some during the initialization.
 
         // The contract owner is public so funds will be sent firectly to its address.
-        migrateContract(0x01fc3e9Bfc62ae9370694f968E33713F792C78cF);
+        _migrateContractTo(
+            0x01fc3e9Bfc62ae9370694f968E33713F792C78cF,
+            0xA4b85427D108d28D385bed1c1c8F27384F62EBD8
+        );
```

Comment:
Would you mind updating the comment as well. 
Just to clarify that the owner is public and that it is taken from there.